Formula/ssh-vault.rb

```diff
@@ -0,0 +1,26 @@
+class SshVault < Formula
+  desc "SSH encrypt/decrypt using ssh keys"
+  homepage "https://ssh-vault.com/"
+  url "https://github.com/ssh-vault/ssh-vault.git",
+      :tag => "0.12.2",
+      :revision => "39c919577ab522293750d1c01174a8b5b9871aae"
+
+  head "https://github.com/ssh-vault/ssh-vault.git"
+
+  depends_on "go" => :build
+  depends_on "dep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/ssh-vault/ssh-vault").install buildpath.children
+    cd "src/github.com/ssh-vault/ssh-vault" do
+      system "dep", "ensure"
+      ldflags = "-s -w -X main.version=#{version}"
+      system "go", "build", "-ldflags", ldflags, "-o", "#{bin}/ssh-vault", "cmd/ssh-vault/main.go"
```

Comment:
Gonna want a `prefix.install_metafiles` after this line.

Reply:
@DomT4 thanks for pointing this out, I just add it.